src/libraries/Common/src/Interop/Unix/System.Native/Interop.MkDir.cs

```diff
@@ -3,12 +3,21 @@
 
 using System;
 using System.Runtime.InteropServices;
+using System.Text;
 
 internal static partial class Interop
 {
     internal static partial class Sys
     {
-        [GeneratedDllImport(Libraries.SystemNative, EntryPoint = "SystemNative_MkDir", CharSet = CharSet.Ansi, SetLastError = true)]
```

Comment:
So since you're now passing a `ValueUtf8Converted`, this means the default value is `CharSet.Unicode`?

Reply:
iirc Windows sys-calls require Unicode while Unix Utf8